src/wagtail_ai/forms.py

```diff
@@ -31,10 +43,21 @@ def clean_prompt(self):
 
         return prompt_uuid
 
-    def errors_for_json_response(self) -> str:
-        errors_for_response = []
-        for _field, errors in self.errors.get_json_data().items():
-            for error in errors:
-                errors_for_response.append(error["message"])
 
-        return " \n".join(errors_for_response)
+class DescribeImageApiForm(ApiForm):
+    image_id = forms.CharField()
+    maxlength = forms.IntegerField(required=False)
```

Comment:
**Suggestion:** Given this is now a user-supplied argument, we should at least limit this to positive numbers only. I can see we could also set a max value of something like `4096` so this is not abused to do something else than populating short descriptions.